nimbus-core/src/main/java/com/antheminc/oss/nimbus/domain/model/config/extension/AbstractConfigEventHandler.java

```diff
@@ -20,18 +20,29 @@
 
 import com.antheminc.oss.nimbus.InvalidConfigException;
 import com.antheminc.oss.nimbus.domain.model.config.ParamConfig;
+import org.springframework.beans.factory.annotation.Autowired;
+import com.antheminc.oss.nimbus.domain.cmd.exec.CommandPathVariableResolver;
+import com.antheminc.oss.nimbus.domain.model.state.EntityState.Param;
 
 
 /**
  * @author Soham Chakravarti
  *
  */
 public abstract class AbstractConfigEventHandler {
+	
```

Comment:
Can we use constructor injection instead of auto wiring ? A pending refactor item for a long time